pybullet_zmq/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
This is the run script from you @hubernikus . Why did you decide to do this manually and not just with `aica-docker`?

Reply:
I always forget the docker tags, as I only use docker a few times every other month. Hence I like to have simple build/run scripts in one place. 
It's a personal preference and probably should not be pushed to the repo.

Reply:
I agree, having it here is good, but if you have the `aica-docker` tools installed, it would make it easier (see comment below)